src/apollo/client.ts

```diff
@@ -13,6 +13,8 @@ export const subgraphClientsUris: { [chainId in SWPRSupportedChains]: string } =
   [ChainId.GOERLI]: 'https://api.thegraph.com/subgraphs/name/dxgraphs/swapr-goerli',
   [ChainId.ARBITRUM_RINKEBY]: 'https://api.thegraph.com/subgraphs/name/dxgraphs/swapr-arbitrum-rinkeby-v2',
   [ChainId.ARBITRUM_GOERLI]: '', // FIXME: fix this once the subgraph is deployed
+  [ChainId.BSC_MAINNET]: 'https://api.thegraph.com/subgraphs/name/unlock-protocol/bsc',
+  [ChainId.BSC_TESTNET]: 'https://api.thegraph.com/subgraphs/name/totemfi/staking-bsc-testnet',
```

Comment:
Swapr core contracts are not deployed on BSC. These records should be empty. Modify `SWPRSupportedChains` to exclude BSC and its testnet.